quill-core/src/main/scala/io/getquill/context/ActionMacro.scala

```diff
@@ -100,7 +100,7 @@ class ActionMacro(val c: MacroContext)
     q"""
       expanded.ast match {
         case io.getquill.ast.Returning(_, _, io.getquill.ast.Property(_, property)) => 
-          property
+          expanded.naming.column(property)
```

Comment:
The renaming should happen here: https://github.com/getquill/quill/blob/master/quill-core/src/main/scala/io/getquill/norm/RenameProperties.scala#L25.

Could you verify why it doesn't work?